src/monorepo-scanner/src/scans/NpmPackages.scan.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import { Workspaces, findMonorepoRoot } from '@adeira/monorepo-utils';
+import path from 'path';
+
+const workspaceLocations = Workspaces.getWorkspacesSync();
+const workspaceMap = new Map();
+const root = findMonorepoRoot();
+
+for (const workspaceLocation of workspaceLocations) {
+  // $FlowAllowDynamicImport
+  const packageJson = require(workspaceLocation);
+  workspaceMap.set(packageJson.name, packageJson);
+}
+
+// $FlowAllowDynamicImport
+const npmPackages = require(path.join(root, 'scripts', 'publishedPackages.json'));
+
+for (const npmPackage of npmPackages) {
+  test(`${npmPackage} has private false flag`, () => {
+    const packageJson = workspaceMap.get(npmPackage);
+    expect(packageJson != null).toGiveHelp('You are trying to release a non existing package');
```

Comment:
`!=` instead `!==` is intentional?

Reply:
Probably it will only be undefined here, so I could change that yes

Reply:
🚀